docker/test/pvs/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN apt-get update --yes \
 ENV PKG_VERSION="pvs-studio-latest"
 
 RUN set -x \
-    && export PUBKEY_HASHSUM="686e5eb8b3c543a5c54442c39ec876b6c2d912fe8a729099e600017ae53c877dda3368fe38ed7a66024fe26df6b5892a" \
+    && export PUBKEY_HASHSUM="ad369a2e9d8b8c30f5a9f2eb131121739b79c78e03fef0f016ea51871a5f78cd4e6257b270dca0ac3be3d1f19d885516" \
```

Comment:
There were some errors when building docker images:
- https://clickhouse-test-reports.s3.yandex.net/28046/da7bbcaab9904896b1d00c5fd7fd60973138e9a8/push_to_dockerhub/dockerbuild.out.31.log
- https://clickhouse-test-reports.s3.yandex.net/28046/da7bbcaab9904896b1d00c5fd7fd60973138e9a8/push_to_dockerhub/dockerbuild.out.41.log

cause of error is `computed checksum did NOT match`

I have downloaded file https://files.viva64.com/etc/pubkey.txt, computed its checksum and updated here accordingly:
```
--2021-09-07 23:04:07--  https://files.viva64.com/etc/pubkey.txt
Resolving files.viva64.com (files.viva64.com)... 172.255.232.179
Connecting to files.viva64.com (files.viva64.com)|172.255.232.179|:443... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://files.pvs-studio.com/etc/pubkey.txt [following]
--2021-09-07 23:04:07--  https://files.pvs-studio.com/etc/pubkey.txt
Resolving files.pvs-studio.com (files.pvs-studio.com)... 173.0.157.39
Connecting to files.pvs-studio.com (files.pvs-studio.com)|173.0.157.39|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 4913 (4,8K) [text/plain]
Saving to: ‘pubkey.txt’

pubkey.txt                                100%[===================================================================================>]   4,80K  --.-KB/s    in 0s      

2021-09-07 23:04:08 (159 MB/s) - ‘pubkey.txt’ saved [4913/4913]

semin-serg@semin-serg-ub:~/Downloads$ sha384sum pubkey.txt 
ad369a2e9d8b8c30f5a9f2eb131121739b79c78e03fef0f016ea51871a5f78cd4e6257b270dca0ac3be3d1f19d885516  pubkey.txt
```